client/my-sites/checkout/controller.jsx

```diff
@@ -44,6 +44,14 @@ import { getProductSlugFromContext, isContextJetpackSitelessCheckout } from './u
 const debug = debugFactory( 'calypso:checkout-controller' );
 
 export function checkoutJetpackSiteless( context, next ) {
+	sitelessCheckout( context, next, { isJetpackCheckout: true } );
+}
+
+export function checkoutAkismetSiteless( context, next ) {
+	sitelessCheckout( context, next, { isAkismetSitelessCheckout: true } );
+}
+
+function sitelessCheckout( context, next, props ) {
```

Comment:
Nitpick, we could rename `props` to `extraProps` or something, since `context` and `next` are technically props too

Reply:
Aha, good idea!